0000-align-iot-subdao-per-packet-cost-with-per-packet-value

```diff
@@ -0,0 +1,35 @@
+# HIP XX: Align IOT subDAO Per Packet Cost with Per Packet Value
```

Comment:
Honestly wondering if this should just named something that's direct. "Double Cost of IoT Network"

Reply:
I wanted to make it more politically palatable!  Gots to think about the yes votes...
But if you want to change the title, that is fine with me.

Reply:
Hah. Feels like not the right thing to focus on. Honest / direct title leads to better results imo.